src/components/Header.js

```diff
@@ -12,4 +14,4 @@ const Header = props => {
   );
 };
 
-export default Header;
+export default connect(state => state)(Header);
```

Comment:
you connected your state to the Header component but are you using it in the component?